package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "disney-villains-api",
+  "version": "1.0.0",
+  "description": "In this two part exrcise you are tasked with create a REST API for accessing Disney Villain data, complete with test suite.",
+  "main": "index.js",
+  "scripts": {
+    "dev": "./node_modules/.bin/nodemon index.js",
+    "lint": "./node_module/.bin/eslint -- format codeframe.",
+    "lint.fix": "./node_module/.bin/eslint -fix -- format codeframe.",
+    "test": "./node_modules/.bin/mocha tests/**/**/.test.js"
```

Comment:
How were you running your tests? This command is broken, the stars are wrong but also your folder is singular `test`.

`"test": "./node_modules/.bin/mocha test/**/*.test.js"`

![image](https://user-images.githubusercontent.com/5775716/81178874-dc1ca880-8f76-11ea-8b96-5f1c0008ee30.png)